concepticondata/conceptlists/Chen-2012-888.tsv

```diff
@@ -388,7 +388,7 @@ Chen-2012-888-386	386	水缸	water tank	695				0	用品工具
 Chen-2012-888-387	387	水桶	bucket	695	2052	BUCKET		561	用品工具
 Chen-2012-888-388	388	罐子	can (jar)	696	3765	STORING POT		557	用品工具
 Chen-2012-888-389	389	水瓢	gourd ladle (wooden dipper)	696	2657	LADLE		0	用品工具
-Chen-2012-888-390	390	火钳	fire tongs	696				566	用品工具
+Chen-2012-888-390	390	火钳	fire tongs	696	272	TONGS		566	用品工具
```

Comment:
The "fire" is part of the compound, so it is a Zange that is used to get things from the fire or similar (don't know for sure). But it is not a generic tong, I'd say (tong is "Zange" in German, right?).